oap-server/server-storage-plugin/storage-elasticsearch7-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch7/query/TraceQueryEs7DAO.java

```diff
@@ -124,8 +125,7 @@ public TraceBrief queryBasicTraces(long startSecondTB,
         }
         sourceBuilder.size(limit);
         sourceBuilder.from(from);
-
-        SearchResponse response = getClient().search(SegmentRecord.INDEX_NAME, sourceBuilder);
+        SearchResponse response = getClient().search(new TimeRangeIndexNameMaker(SegmentRecord.INDEX_NAME, startSecondTB, endSecondTB), sourceBuilder);
```

Comment:
I think ES7 and ES6 should use different TimeRangeIndexNameMaker, due to different package name or something like that. Please recheck.

Reply:
![image](https://user-images.githubusercontent.com/5441976/90249674-78572500-de6d-11ea-86f9-9c039504c1af.png)

You could find this in the e2e logs. And es7 e2e is going to fail because of the error.

Reply:
Notice in the current codes, `ElasticSearch7Client` and `ElasticSearchClient` have their own `#search` methods, your new `#doSearch` method has not been included in the ES 7 client.

Reply:
today i test the codes in our test env  and find the bug,already fix it ,and pass the tests

Reply:
waiting to  pass all ci proesses

Reply:
This is a very tricky bug due to Elastic changed the package name, but with the same class name.

Reply:
yes,the error is no such method exception when testiing,i realize may be jars conflict.but took so many time to  find it

Reply:
Finally, we get this method working, and it should be helpful to many users.